Source/WarpX.H

```diff
@@ -1212,6 +1212,20 @@ protected:
     //! This function is called in amrex::AmrCore::InitFromScratch.
     void PostProcessBaseGrids (amrex::BoxArray& ba0) const final;
 
+    //! If this is true, AMReX's RRSFC strategy is used to make
+    //! DistributionMapping in the virtual function MakeDistributionMap. The
+    //! default is false. Note that the DistributionMapping made by the
+    //! MakeDistributionMap function could still be overridden by load
+    //! balancing. In the RRSFC strategy, the Round robin method is used to
+    //! distribute Boxes orderd by the space filling curve. This might help
+    //! avoid some processes running out of memory due to having too many
+    //! particles during initialization.
+    static bool roundrobin_sfc;
+
```

Comment:
Let's move this comment block as the doxygen block after the title of `MakeDistributionMap` or inside its body before the `if`?

I would just query the `ParmParse` here again instead of creating a static member variable.
```suggestion
```